frontend/views/components/GroupSettings/InviteLinks.vue

```diff
@@ -0,0 +1,332 @@
+<template lang="pug">
+page-section.c-section(:title='L("Invite links")')
+  template(#cta='')
+    .select-wrapper.c-select-wrapper(:class='{ focus: ephemeral.selectbox.focused }')
+      select.button.is-small.is-outlined.c-select(
+        ref='select'
+        v-model='ephemeral.selectbox.selectedOption'
+        @change='unfocusSelect'
+      )
+        option(value='Active') {{ L('Active links') }}
+        option(value='All') {{ L('All links') }}
+      span.c-rect
+
+  i18n.has-text-1.c-invite-description(tag='p') Here's a list of all invite links you own
+
+  table.table.c-table(v-if='activeList.length')
+    thead
+      tr
+        i18n.c-name(tag='th') created for
+        i18n.c-invite-link(tag='th') invite link
+        i18n.c-state(tag='th') state
+        th.c-action(
+          :aria-label='L("action")'
+        )
+    tbody
+      tr(
+        v-for='(item, index) in activeList'
+        :key='index'
+      )
+        td.c-name
+          | {{ item.name }}
+          tooltip.c-name-tooltip(
+            direction='top'
+            :isTextCenter='true'
+            :text='L("This invite link was only available during the onboarding period.")'
+          )
+            button.is-icon-small(v-if='item.isAnyoneLink')
+              i.icon-info-circle
+        td.c-invite-link
+          copy-to-clipboard.c-invite-link-wrapper(:textToCopy='item.inviteLink')
+            span.link.has-ellipsis {{ item.inviteLink }}
+            button.is-icon-small.has-background.c-invite-link-button
+              i.icon-copy.is-regular
+        td.c-state
+          i18n.c-state-description {{ item.state.description }}
+          i18n.c-state-expire(
+            v-if='item.state.expireInfo'
+            :class='item.state.isExpired ? "expired" : ""'
+          ) {{ item.state.expireInfo }}
+        td.c-action
+          invite-action-button.c-invite-action-button(:linkExpired='item.state.isExpired')
+  .c-empty-list(v-else)
+    SvgInvitation
+
+  // TODO: router-link doesn't work and it might be related to the fact that this p tag is the output result of i18n.vue component.
+  // Discussion needed to solve the problem.
+  i18n.c-invite-footer(
+    tag='p'
+    html='To generate a new link, you need to <router-link class="link" to="/invite">propose adding a new member</router-link> to your group.'
+  )
```

Comment:
Once #727 is merged into `master`, merge `master` and you'll be able to use `<router-link>`. Just make sure to move the text out of `html`, and place the HTML into `:args` as shown in the example in #727

Reply:
fixed! thanks!